app/src/main/java/com/codepath/travel/activities/CreateStoryActivity.java

```diff
@@ -147,6 +171,38 @@ private void setUpRecyclerView() {
         mItemTouchHelper.attachToRecyclerView(rvStoryPlaces);
     }
 
+    private void addSelectedPlaceInTrip(Place place, String photoReference) {
+        StoryPlace storyPlace = new StoryPlace(mNewTrip, place);
+        storyPlace.setThumbnail(photoReference);
```

Comment:
Yeah I don't see why we would need different photos for a cover vs thumbnail. We basically just need one representative photo for a story place. photoUrl sounds good. thanks!